parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -415,16 +377,13 @@ cleanupRetaggedImages() {
 }
 
 cleanUpContainerImages() {
-    # run cleanUpHyperkubeImages and cleanUpKubeProxyImages concurrently
     export KUBERNETES_VERSION
     export CLI_TOOL
     export -f retrycmd_if_failure
     export -f removeContainerImage
     export -f cleanUpImages
-    export -f cleanUpHyperkubeImages
     export -f cleanUpKubeProxyImages
-    bash -c cleanUpHyperkubeImages &
-    bash -c cleanUpKubeProxyImages &
+    bash -c cleanUpKubeProxyImages
```

Comment:
are we intentionally removing & so cleanUpKubeProxyImages no longer runs in the background?

Reply:
Thanks for bringing this up. It could've slowed down CSE, so I reverted it back to the background run.